README.md

```diff
@@ -272,8 +264,8 @@ poetry env use /usr/local/bin/python3
    * Example from Ubuntu 22.04 -- adapt to your own Linux distro. Python should
      be version 3.11 or above, and GG has been tested in versions up to 3.13.
      ```bash
-     sudo apt install python3.11 python3-pip python3-tk idle-python3.11 git
-     sudo python3.11 -m pip install poetry
+     sudo apt install python3.11 python3-pip python3-tk idle-python3.11 git pipx
+     sudo pipx install poetry
```

Comment:
In line 272 below, you say
> * If this does not work, try using `pipx` in place of `pip` in the above commands.

Do you need that anymore, since you change the command to `pipx`?

Reply:
Good catch; fixed that by deleting that line.